fuzz/fuzz_targets/fuzz_regex_match.rs

```diff
@@ -0,0 +1,19 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &[u8]| {
+    if let Ok(data) = std::str::from_utf8(data) {
+        // split data into regular expression and actual input to search through
+        use std::cmp::max;
+        let len = data.chars().count();
+        let split_off_point = max(len / 5, 1) as usize;
```

Comment:
What if this split point was determined by the first byte or two of the data so the fuzzer the regex and input size were not locked to one another?

Reply:
Good idea - how does the new one look?

Reply:
looks good to me, but it's only @BurntSushi's opinion that matters here.